addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductItem.js

```diff
@@ -40,10 +40,6 @@ odoo.define('point_of_sale.ProductItem', function(require) {
                 return formattedUnitPrice;
             }
         }
-        async onProductInfoClick() {
-            const info = await this.env.pos.getProductInfo(this.props.product, 1);
-            this.showPopup('ProductInfoPopup', { info: info , product: this.props.product });
-        }
```

Comment:
removing dead code that should have been removed due to https://github.com/odoo/odoo/pull/99033